docs/docusaurus/docs/gx_cloud/tutorials/getting_started/cloud_quickstart.md

```diff
@@ -0,0 +1,305 @@
+
+### 3. Create Expectations
+
+#### 3.1 Create Expectation Suite
+
+An <TechnicalTag tag="expectation_suite" text="Expectation Suite"/> is a collection of verifiable assertions about data. Paste this snippet into Jupyter Notebook to create it:
+
+```python title="Jupyter Notebook"
+expectation_suite = context.create_expectation_suite(
+    expectation_suite_name=None # Enter your expectation suite name here
+)
+```
+
+#### 3.2 Add Expectation
+
+Modify and run this snippet to add an <TechnicalTag tag="expectation" text="Expectation"/> to <TechnicalTag tag="expectation_suite" text="Expectation Suite"/> you just created:
+
+```python title="Jupyter Notebook"
+# Get an existing Expectation Suite
+expectation_suite_id = expectation_suite.ge_cloud_id
+expectation_suite = context.get_expectation_suite(ge_cloud_id=expectation_suite_id)
+
+# Look up all expectations types here - https://greatexpectations.io/expectations/
+expectation_configuration = gx.core.ExpectationConfiguration(**{
+  "expectation_type": "expect_column_values_to_not_be_null",
+  "kwargs": {
+    "column": None, # Enter your column name here
+  },
+  "meta":{}
+})
+
+expectation_suite.add_expectation(
+    expectation_configuration=expectation_configuration
+)
+print(expectation_suite)
+
+# Save the Expectation Suite
+context.save_expectation_suite(expectation_suite)
+```
+
+With the Expectation defined above, we are stating that we _expect_ the column of your choice to always be populated. That is: none of the column's values should be null.
+
+
+### 4. Validate data
+
+#### 4.1 Create Checkpoint
+
+Now that we have connected to data and defined <TechnicalTag tag="expectation" text="Expectation"/>, it is time to validate whether our data meets the expectation. To do this, we define a <TechnicalTag tag="checkpoint" text="Checkpoint"/> (which will allow us to repeat the <TechnicalTag tag="validation" text="Validation"/> in the future).
+
+```python title="Jupyter Notebook"
+checkpoint_name = None  # name your checkpoint here
+
+# uncomment the lines below after successfully creating your Checkpoint to run this code again!
+# checkpoint = context.get_checkpoint(checkpoint_name)
+# checkpoint_id = checkpoint.ge_cloud_id
+
+checkpoint_config = {
+  # "id": checkpoint_id,  # uncomment after successfully creating your Checkpoint
+  "name": checkpoint_name,
+  "validations": [{
+      "expectation_suite_name": expectation_suite.expectation_suite_name,
+      "expectation_suite_ge_cloud_id": expectation_suite.ge_cloud_id,
+      "batch_request": {
+          "datasource_name": "YOUR DATASOURCE NAME HERE YOU WANT TO CONNECT TO",
+          "data_connector_name": "YOUR DATA CONNECTOR NAME HERE YOU WANT TO CONNECT TO",
+          "data_asset_name": "SET YOUR DATA ASSET NAME HERE",
+      },
```

Comment:
See above comments about setting variables and just referencing them again later, so that user just has to set once.